keps/sig-network/20190324-remove-kube-proxy-autocleanup.md

```diff
@@ -0,0 +1,74 @@
+---
+kep-number: 0
+title: Remove kube-proxy's automatic clean up logic
+authors:
+  - "@bowei"
+owning-sig: sig-network
+participating-sigs:
+reviewers:
+  - "@andrewsykim"
+approvers:
+  - "@bowei"
+  - "@thockin"
+editor:
+creation-date: 2018-03-24
+last-updated: 2018-03-24
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Remove kube-proxy's automatic clean up logic
+
+## Summary
+
+Remove the automatic network rule cleanup from kube-proxy.
+Only clean up rules on boot if the `--cleanup` flag is present.
+
+## Motivation
+
+kube-proxy's rule cleanup has a substantial performance impact,
+and is not normally necessary.
+
+### Goals
+
+* Reduce the typical kube-proxy startup time,
```

Comment:
Avoid bugs where different proxier modules' rules overlap, causing user-visible impact between cleanup and full sync